torchbox/torchbox.com
# Link underline colour

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1472020372)

### Description of Changes Made

- Update all links to have an underline of ` rgba(255, 255, 255, 0.5)` or `#989b9b`
- Don't apply this to navigation links in the dropdowns as the new colour does not pass colour contrast - stick with the colour of the text
- Update some instances where the `link-styles()` mixin was not used

### How to Test

- Browse the site and make sure all link underlines look consistent
- Might be easier to test components on the styleguide page in the pattern library

### Screenshots

There's too many components that this has changed to provide screenshots.

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
